extract.js

```diff
@@ -43,51 +44,47 @@ function generateFeatures(url, client, clientid) {
     // clientFeatures are the same for all peerconnections but are saved together
     // with each peerconnection anyway to make correlation easier.
     const clientFeatures = {};
-    Object.keys(features).forEach(fname => {
-        if (features[fname].length === 1) {
-            let feature = features[fname].apply(null, [client]);
-            if (feature !== undefined) {
-                if (typeof feature === 'object') {
-                    Object.keys(feature).forEach(subname => {
-                        feature[subname] = safeFeature(feature[subname]);
-                        if (!isProduction) {
-                            console.log('PAGE', 'FEATURE', fname + capitalize(subname), '=>', safeFeature(feature[subname]));
-                        }
-                        clientFeatures[fname + capitalize(subname)] = feature[subname];
-                    });
-                }  else {
-                    feature = safeFeature(feature);
+    Object.keys(clientfeatures).forEach(fname => {
```

Comment:
hard to read having a variable clientfeatures and one clientFeatures

Reply:
agree but i'd separate renames from reorgs. One change at a time...